README.md

```diff
@@ -0,0 +1,60 @@
+# Covid-19-Tracker
+
+>This React capstone project is about building a mobile web application to check a list of affected countries by covid19 created by making use of React and Redux.
+
+## Built With
+
+- React, 
+- Redux,
+- CSS3
+
+Major language
+- Javascript
+
+
+## Live demo
+[Live demo](https://aquamarine-concha-9215bd.netlify.app/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+By running this project loccally, you will be required to have:
+
+- A computer having diffents browsers
+- VSCode, node installed
+- Clone the app by running `git clone https://github.com/keza681/covid-19-tracker.git`
+- Few knowledge about Node package module(npm) to start the app.
+- Run `npm install` to install automatically the dependencies.
+- Run `npm start` after installing dependencies in order to run the app
+- Run `npm test` to test the project rendenring components.
+- Run `npm build` to build the project for deploying.
+
+## Authors
+
+👤 **Keza**
+
+- GitHub: [Visit my github](https://github.com/keza681)
+- LinkedIn: [Linkedin Profile](https://www.linkedin.com/in/linda-keza-a10150218/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome😉
+
+
+## Show your support
+
+Leave a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+
+## 📝 License
+
+MIT License
+
+
```

Comment:
### OPTIONAL
I wrote it as an optional but this is a requirement that is not mentioned in the first review. Please fix it.
- [ ] You should add a creative common license
- [ ] You should add design link in acknowledgment section.
[Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Original%20design%20idea,of%20your%20project.)